### DISCUSSION TITLE
Error setting up a new profile

### DISCUSSION THREAD

**Discussion by iHumberto** [microsoft/terminal#13669] [Q&A] [answered]
Hi guys!

I don't know if this is the right place for this, but, I have a Raspberry Pi 4 which I'm using with pihole and other stuff. I created a new profile to with ssh to open directly the RPI terminal (I'm using public/private key, so I don't have to input password), and this works fine.

I installed Docker at this RPI and downloaded Kali Linux image. Now I want to add a new profile to open RPI and go to Kali just by clicking at this new profile but it gives me an error message.

 The RPI profile that works fine has this options:

```
 {
                "commandline": "ssh pi@raspberrypi",
                "guid": "{e7625667-dd68-5bc0-8ecd-de3b7612cfcc}",
                "icon": "C:/Users/humbe/raspberrypi.png",
                "name": "RaspberryPi",
                "tabTitle": "RaspberryPi"
            },
```

So I just copied this and add "docker run --tty --interactive kalilinux/kali-rolling" but gives the message "_the input device is not a TTY_".

I already tried to add sleep _(the error message is: "sleep: option not recognized '--tty'")_ , timout _(the error says: "the input device is not a TTY")_ and && _(the error message says: "sintax error nearby unexpected token '&&'")_ to the command line option, but also doesn't work. 

I also tried to create an bash script, put it at my home directory and put the command " ./docker.sh" at the command line option, but also gives me the same error messages . 

I'm completly lost. There is anyway to do this?

**Comment by ianjoneill** [comment, marked as answer]
If a command to run is added to an SSH command line, it's run non-interactively by default. You want to add `-t` to your SSH command to force a TTY allocation.

i.e. make your command line: `ssh -t pi@raspberrypi docker run --tty --interactive kalilinux/kali-rolling`.

See the SSH manual: https://man.openbsd.org/ssh
